cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/schedule/SchedulerServiceTest.java

```diff
@@ -54,7 +67,11 @@ public static void finish() {
   @Test
   public void testSchedulesAcrossNamespace() throws Exception {
     AppFabricTestHelper.deployApplication(AppWithWorkflow.class);
+    ApplicationSpecification applicationSpecification = store.getApplication(appId);
+
     schedulerService.schedule(program, programType, ImmutableList.of(schedule1));
+    applicationSpecification = createNewSpecification(applicationSpecification, program, programType, schedule1);
```

Comment:
not sure I understand these changes. Are we not able to add a schedule to an existing app any more?